lahja/common.py

```diff
@@ -52,7 +56,15 @@ class BaseEvent:
 
     _origin = ""
     _id: Optional[str] = None
-    _config: Optional[BroadcastConfig] = None
+
+    is_bound = False
+
+    def bind(self, endpoint: "EndpointAPI", id: Optional[str]) -> None:
+        if self.is_bound:
+            raise RuntimeError("Event is already bound")
```

Comment:
Not a strong opinion but maybe worth following our standard practice to use a subclass of `LahjaError` here.